mantidproject/mantid
# Add weighted ls fitting system tests.

Fixes #13348
### To test
- check that systemtest pass
- check that text in systemtest reads well and is clear
- check 1st good example added to this test
